### ISSUE TITLE
Fix pytest 4.x warning-as-failure: RemovedInPytest4Warning: MarkInfo objects are deprecated... in test_tracking_protection_shield

### ISSUE THREAD

**Issue by stephendonner** [mozilla/FoxPuppet#222]
Thankfully this isn't (yet) on master, but the PR for the bump to pytest 4.x yielded this gem:

```
==================================== ERRORS ====================================
______________ ERROR at setup of test_tracking_protection_shield _______________
request = <SubRequest 'firefox_options' for <Function 'test_tracking_protection_shield'>>
firefox_path = None, firefox_profile = None
    @pytest.fixture
    def firefox_options(request, firefox_path, firefox_profile):
        options = Options()
    
        if firefox_profile is not None:
            options.profile = firefox_profile
    
        if firefox_path is not None:
            options.binary = FirefoxBinary(firefox_path)
    
        args = request.node.get_marker('firefox_arguments')
        if args is not None:
            for arg in args.args:
                options.add_argument(arg)
    
        prefs = request.node.get_marker('firefox_preferences')
        if prefs is not None:
>           for name, value in prefs.args[0].items():
E           RemovedInPytest4Warning: MarkInfo objects are deprecated as they contain merged marks which are hard to deal with correctly.
E           Please use node.get_closest_marker(name) or node.iter_markers(name).
E           Docs: https://docs.pytest.org/en/latest/mark.html#updating-code
.tox/py27/lib/python2.7/site-packages/pytest_selenium/drivers/firefox.py:88: RemovedInPytest4Warning
------------------------------- pytest-selenium --------------------------------
Driver log: /tmp/pytest-of-travis/pytest-0/test_switch_to0/driver.log
- generated html file: /home/travis/build/mozilla/FoxPuppet/results/py27.html --
```

**Comment by davehunt** [comment]
The dependencies label is managed by Dependabot, and used to indicate that a pull request updates a dependency.

**Comment by jrbenny35** [comment]
Fixed in #236
